### PR TITLE
Modified CalculateDepth to get coverage on whole alignment adam files

### PR DESCRIPTION
**Pull request by akmorrow13**

### CODE REVIEW

File: adam-cli/pom.xml [hunk 1]
```diff
@@ -141,6 +141,12 @@
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_2.10</artifactId>
+      <version>${spark.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-sql_2.10</artifactId>
+      <version>${spark.version}</version>
```
Comment (fnothaft) [inline adam-cli/pom.xml]: Delete here.

File: pom.xml [hunk 2]
```diff
@@ -440,6 +440,10 @@
         </exclusions>
       </dependency>
       <dependency>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-sql_2.10</artifactId>
+      </dependency>
```
Comment (fnothaft) [inline pom.xml]: Add `<version>${spark.version}</version>` here.

File: adam-cli/src/main/scala/org/bdgenomics/adam/cli/CalculateDepth.scala [hunk 3]
```diff
@@ -53,64 +52,83 @@ class CalculateDepthArgs extends Args4jBase with ParquetArgs {
   @Argument(required = true, metaVar = "ADAM", usage = "The Read file to use to calculate depths", index = 0)
   val adamInputPath: String = null
 
-  @Argument(required = true, metaVar = "VCF", usage = "The VCF containing the sites at which to calculate depths", index = 1)
+  @Argument(required = true, metaVar = "COVERAGE", usage = "Location to save coverage to parquet file", index = 1)
```
Comment (fnothaft) [inline adam-cli/src/main/scala/org/bdgenomics/adam/cli/CalculateDepth.scala]: Since `-print` is an option as well, this should be an option.

**Comment by fnothaft** [comment]
Since this is a major modification of `CalculateDepth`, perhaps we should move this/calculate depth into two separate commands? That would be my preference.

File: adam-cli/src/main/scala/org/bdgenomics/adam/cli/CalculateDepth.scala [hunk 4]
```diff
@@ -53,64 +52,83 @@ class CalculateDepthArgs extends Args4jBase with ParquetArgs {
...
+            count
+          ))
+      }
     }
+
+    /* Save coverage to output path */
+    val sqlContext = new org.apache.spark.sql.SQLContext(sc)
+
+    val frequencies = sqlContext.createDataFrame(depths.map(r => Coverage(r._1.referenceName, r._1.start, r._2)))
+    frequencies.write.parquet(args.outputPath)
```
Comment (akmorrow13) [inline adam-cli/src/main/scala/org/bdgenomics/adam/cli/CalculateDepth.scala]: let's save this as a feature

**Comment by fnothaft** [comment]
@akmorrow13 I'm going to close this in favor of https://github.com/bigdatagenomics/quinine/issues/14.

**Pull request closed without merging**